packages/react-reconciler/src/ReactFiberReconciler.js

```diff
@@ -10,6 +10,7 @@
 import type {Fiber} from './ReactFiber';
 import type {FiberRoot} from './ReactFiberRoot';
 import type {RootTag} from 'shared/ReactRootTags';
+import type {TouchedViewDataAtPoint} from 'react-native-renderer/src/ReactNativeTypes';
```

Comment:
This import is causing CI to fail.

Reply:
We can't have dependencies on RN from the reconciler bundle. The folders are checked independently.

Also from a layering perspective it's probably not right.

Reply:
I'm a little confused that some of the commits seemed to pass CI